src/client/java/teammates/client/scripts/ListActiveInstructors.java

```diff
@@ -0,0 +1,54 @@
+package teammates.client.scripts;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.storage.entity.FeedbackSession;
+
+/**
+ * Script to generate emails of active instructors within a period.
+ */
+public class ListActiveInstructors extends RemoteApiClient {
+    @Override
+    protected void doOperation() {
+        //2020/01/01
+        int startDate = 1;
+        int startMonth = 1;
+        int startYear = 2020;
+        //2021/02/01
+        int endDate = 1;
+        int endMonth = 2;
+        int endYear = 2021;
+
+        LocalDate startPoint = LocalDate.of(startYear, startMonth, startDate);
+        LocalDate endPoint = LocalDate.of(endYear, endMonth, endDate);
+        long startTime = startPoint.atStartOfDay().toInstant(ZoneOffset.UTC).toEpochMilli();
+        long endTime = endPoint.atStartOfDay().toInstant(ZoneOffset.UTC).toEpochMilli();
+
+        ArrayList<String> activeInstructorEmails = new ArrayList<>();
```

Comment:
I think `Set` would be a more appropriate data structure for `contains` and `add` operations?